<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DABBUSINGH" failures="1" tests="3" name="Test" time="0.059" errors="0" timestamp="2024-11-27T10:32:06 IST">
  <testcase classname="Test.firstClass" name="testFail" time="0.016">
    <failure type="java.lang.AssertionError" message="This test will fail expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: This test will fail expected [true] but found [false]
at Test.firstClass.testFail(firstClass.java:16)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- testFail -->
  <testcase classname="Test.firstClass" name="testPass" time="0.002"/>
  <testcase classname="Test.firstClass" name="testSkip" time="0.002">
    <skipped/>
  </testcase> <!-- testSkip -->
</testsuite> <!-- Test -->
